src/components/FormValidator.js

```diff
@@ -79,4 +79,12 @@ export default class FormValidator {
 
```

Comment:
Please, delete `this.disableBtn();` from line 77 from `FormValidator.js`

You need to `disable` the button only after **resetting** the form in block `then` otherwise the user will not be able to click it if there is a server error